### ISSUE TITLE
Build failing

### ISSUE THREAD

**Issue by jpdefillippo** [quickstonesw/Appcelerator-Urban-Image-Library#1]
Hey,

I'm trying to build the module per the instructions but it fails with this message:

/Users/jpd/Desktop/quickstonesw-Appcelerator-Urban-Image-Library-fb886f1/Classes/QsUrbanimageLibraryModule.h:16:9: fatal error: 'TiModule.h' file not found [2]
 #import "TiModule.h"
         ^
1 error generated.

*\* BUILD FAILED **

The following build commands failed:
    CompileC build/urbanimagelibrary.build/Release-iphoneos/urbanimagelibrary.build/Objects-normal/armv7/QsUrbanimageLibraryModule.o Classes/QsUrbanimageLibraryModule.m normal armv7 objective-c com.apple.compilers.llvm.clang.1_0.compiler
    CompileC build/urbanimagelibrary.build/Release-iphoneos/urbanimagelibrary.build/Objects-normal/armv6/QsUrbanimageLibraryModule.o Classes/QsUrbanimageLibraryModule.m normal armv6 objective-c com.apple.compilers.llvm.clang.1_0.compiler
(2 failures)
xcodebuild failed

Any help is appreciated.

**Comment by TanelTeemusk** [comment]
jpdefilippo

I got it working just fine. Had the same problem as you. Apparently you are trying to compile the module against a titanium sdk that you don't have. This build tries to access titanium sdk 1.8.1 
to compile it against other versions you will have to modify the titanium.xcconfig file to match your sdk:
I'm using 1.8.0 so I changed the line on top to:
TITANIUM_SDK_VERSION = 1.8.0

then it compiled the zip for me and build succeeded.

cheers.

Tanel Teemusk
http://teemusk.com
